assets/queries/cloudFormation/hardcoded_aws_access_key_in_lambda/query.rego

```diff
@@ -0,0 +1,37 @@
+package Cx
+
+CxPolicy[result] {
+	document := input.document[i]
+	resource := document.Resources[key]
+	resource.Type == "AWS::Lambda::Function"
+	properties := resource.Properties
+
+	envVars := properties.Environment.Variables
+	re_match("[A-Za-z0-9/+=]{40}", envVars[var])
```

Comment:
Instead of two rules for this query, create array with both regex:
```suggestion
	regexAccessKey := ["[A-Za-z0-9/+=]{40}", "[A-Z0-9]{20}"]
	some var
	re_match(regexAccessKey[_], envVars[var])
```